src/Monitor.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using System.Linq;
 using SharpPcap;
-using SharpPcap.Npcap;
+using SharpPcap.LibPcap;
```

Comment:
I haven't looked too closely into the SharpPcap breaking changes, but is it safe to say that Npcap is no longer supported? Or is there another reason you switched to LibPcap?

Reply:
Correct, Npcap is gone.  LibPcap is similar but now a direct replacement.